src/components/atoms/ReviewCardCarousel/ReviewCard.tsx

```diff
@@ -0,0 +1,56 @@
+import Image from "next/image";
+import React, { FC } from "react";
+import styles from "./ReviewCard.module.scss";
+import StarRating from "./StarRating";
+
+type ReviewCard = {
+  reviewer: string;
+  comment: string;
+  image: string;
+  rate: number;
+  isActive: boolean;
+};
+
+const ReviewCard: FC<ReviewCard> = ({
+  reviewer,
+  comment,
+  image,
+  rate,
+  isActive,
+}) => {
+  const capitalizeFirstLetterOfEachWord = (str: string) => {
```

Comment:
We can do the same without javascript but using ` text-transform: capitalize;`. It is easy, understandable, and clean